lib/testPaymentExpiry.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+const _ = require('lodash')
+const moment = require('moment')
+const request = require('co-request')
+const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
+
+module.exports = function * (config, payment) {
+  let tester = new TransferTester(config, payment)
+  yield tester.loadCaseExpiries()
+  return tester
+}
+
+function TransferTester (config, payment) {
+  this.config = config
+  this.source_transfers = payment.source_transfers
```

Comment:
The "single module" would have to be reused across fb-{shared,connector,ledger,notary,sender,pathfind,demo} in order to keep the code consistent.

That solution is needlessly complex. The current naming scheme has the substantial perk of mapping directly to -shared's schemas, and requires no additional abstractions. The only downside is one of aesthetics, which is personal preference anyhow.

I think the best option is to leave it as-is.